src/components/utils/useOutsideClick/index.ts

```diff
@@ -0,0 +1,2 @@
+export {useOutsideClick} from './useOutsideClick';
+export type {UseOutsideClickProps} from './useOutsideClick';
```

Comment:
Let's also reexport this file in main index

Reply:
done